webapp/src/components/confirm_modal.tsx

```diff
@@ -92,10 +92,11 @@ export default class ConfirmModal extends PureComponent<Props, State> {
         return nextProps.show !== this.props.show;
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
-        if (this.props.show && !nextProps.show) {
+    // UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
+    componentDidUpdate(prevProps) { // eslint-disable-line camelcase
```

Comment:
Please remove the commented out line here. Also we don't need the eslint disable line here anymore.

Reply:
Done